### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Ivam2280**
- [DEMO LINK](https://Ivam2280.github.io/layout_catalog/)
- [TEST REPORT LINK](https://Ivam2280.github.io/layout_catalog/report/html_report/)

- [ ] All component follow BEM and use SCSS
- [ ] repaeted sizes and special colors are put to variables
- [ ] Grid is used for the columns
- [ ] cards are shown in 1, 2, 3 or 4 columns based on screen resolution
- [ ] All changes on `:hover` are smooth
- [ ] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -22,6 +22,393 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__wrap">
```
Comment (volodymyr-soltys97) [inline src/index.html:26]: This div is redundant
```suggestion
```

File: src/index.html [hunk 2]
```diff
@@ -22,6 +22,393 @@
...
+        <a href="/">
+          <img
+            class="header__logo"
+            src="./images/logo.png"
+            alt="logo"
+          />
+        </a>
+        <nav class="nav">
```
Comment (volodymyr-soltys97) [inline src/index.html:34]:
```suggestion
        </a>
        
        <nav class="nav">
```

File: src/index.html [hunk 3]
```diff
@@ -22,6 +22,393 @@
...
+          <ul class="nav__list">
+            <li>
```
Comment (volodymyr-soltys97) [inline src/index.html:36]: Add classes everywhere
```suggestion
            <li class="nav__item">
```

File: src/index.html [hunk 4]
```diff
@@ -22,6 +22,393 @@
...
+                href="/"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="main">
```
Comment (volodymyr-soltys97) [inline src/index.html:112]:
```suggestion
    </header>
    
    <main class="main">
```

File: src/index.html [hunk 5]
```diff
@@ -22,6 +22,393 @@
...
+        </div>
+        <a
+          class="card__link"
+          href="#"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
+      <div class="card">
```
Comment (volodymyr-soltys97) [inline src/index.html:153]: Add the blank line between the cards
```suggestion
      </div>
      
      <div class="card">
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code

**Review by lerastarynets — APPROVED** [review]
lgtm 🚀

**Pull request closed without merging**